bump-version/action.yml

```diff
@@ -0,0 +1,46 @@
+name: Bump Version
+description: Bump the version and push if not a dry-run
+inputs:
+  version:
+    description: The new version
+    required: true
+  version_bump_script:
+    description: The version bump script
+    required: true
+  commit_template:
+    description: The template for the git commit message
+    default: "BUMP ${VERSION}"
+  dry_run:
+    description: Whether this is a dry run
+    required: true
```

Comment:
How about naming this `push_commit` since that's what it does?

Context: in the PHP repositories, I don't want to push right away since we merge the changes to another branch so our merge-up automation doesn't pick up these changes (as they would conflict anyways)

Reply:
Done